subprojects/gradle/build-verdict/src/test/kotlin/com/avito/android/build_verdict/BuildVerdictPluginTest.kt

```diff
@@ -0,0 +1,194 @@
+package com.avito.android.build_verdict
+
+import com.avito.android.build_verdict.BuildFailureListener.Companion.buildVerdictFileName
+import com.avito.test.gradle.TestProjectGenerator
+import com.avito.test.gradle.dir
+import com.avito.test.gradle.gradlew
+import com.avito.test.gradle.kotlinClass
+import com.avito.test.gradle.module.AndroidAppModule
+import com.google.common.truth.Truth
+import com.google.gson.GsonBuilder
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import java.io.File
+
+class BuildVerdictPluginTest {
+
+    @field:TempDir
+    lateinit var temp: File
+    private val gson = GsonBuilder().create()
+
+    @Test
+    fun `When kotlin compilation fails Then build-verdict contains kotlin compile info`() {
```

Comment:
name should be on lower-case

Reply:
Why? As for me, it's easier to read function when it has anchors for the eyes

Reply:
it was in https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules
and we chose ` - ` for anchors

Reply:
>  it's easier to read function when it has anchors for the eyes

The question is not about naming. It is about changing agreements.
In all other tests, we have the same convention. If we want to change it, this is fine. But it's better to do it explicitly and fix it somewhere.